### PR TITLE
TestNG - changes.

### PR DESCRIPTION
**Pull request by Radu-Stroe**
Change several outdated WebElements - Homepage.
Change several outdated WebElements, create a loginMenu WebElement - LoginPage.
Change myNewsLettersButton WebElement - MyAccountPage.
Change several outdated WebElements, change order of objects - NavigationPage.
Change myDetailsButton WebElement - PersonalDetailsPage.
Change several outdated WebElements - RegisterCompletePage.
Set priority for tests.

### CODE REVIEW

File: src/test/java/internship/tests/PersonalDetailsTest.java [hunk 1]
```diff
@@ -6,19 +6,18 @@
...
 
 import java.util.concurrent.TimeUnit;
 
 public class PersonalDetailsTest extends TestBase {
-    public static ThreadLocal<SoftAssert> softAssert = new ThreadLocal<>();
-    private final String TEST_EMAIL = "radu.telegraph+11@gmail.com";
+    private final String TEST_EMAIL = "radu.telegraph+10@gmail.com";
     private final String TEST_PASSWORD = "qwer1234";
-    private final String TEST_NEW_EMAIL = "radu.telegraph+12@gmail.com";
+    private final String TEST_NEW_EMAIL = "radu.telegraph+11@gmail.com";
```
Comment (cristinanedin) [inline src/test/java/internship/tests/PersonalDetailsTest.java]: this doesn't look maintainable

File: src/test/java/internship/tests/NavigationPageTest.java [hunk 2]
```diff
@@ -11,11 +11,10 @@
 
     @Test(testName = "Open Navigation page",
             description = "Open Navigation page and verify sections",
-            dataProvider = "sectionNavigation", dataProviderClass = DataProviders.class)
+            dataProvider = "sectionNavigation", dataProviderClass = DataProviders.class,
+            priority = 2)
```
Comment (cristinanedin) [inline src/test/java/internship/tests/NavigationPageTest.java]: why you need this?

**Review by cristinanedin — CHANGES_REQUESTED** [review]

File: src/test/java/internship/tests/PersonalDetailsTest.java (hunk 1 above)
Comment (Radu-Stroe) [reply src/test/java/internship/tests/PersonalDetailsTest.java]: Noted and fixed.

File: src/test/java/internship/tests/NavigationPageTest.java (hunk 2 above)
Comment (Radu-Stroe) [reply src/test/java/internship/tests/NavigationPageTest.java]: It seems that I don't need priority for my tests. Fixed.

**Review by cristinanedin — APPROVED** [review]

**Pull request merged**